content/blog/integrating-devops-and-security-for-scalable-platform-engineering/index.md

```diff
@@ -0,0 +1,202 @@
+meta_desc: Gain insights from industry leaders on integrating DevOps and security in platform engineering to build scalable, secure, and developer-friendly platforms.
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - sara-huddleston
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - devsecops
+    - platform-engineering
+    - devex
+    - devops
+    - security
+
+
+# The social copy used to promote this post on Twitter and Linkedin. These
+# properties do not actually create the post and have no effect on the
+# generated blog page. They are here strictly for reference.
+
+# Here are some examples of posts we have made in the past for inspiration:
+# https://www.linkedin.com/feed/update/urn:li:activity:7171191945841561601
+# https://www.linkedin.com/feed/update/urn:li:activity:7169021002394296320
+# https://www.linkedin.com/feed/update/urn:li:activity:7155606616455737345
+# https://twitter.com/PulumiCorp/status/1763265391042654623
+# https://twitter.com/PulumiCorp/status/1762900472489185492
+# https://twitter.com/PulumiCorp/status/1755637618631405655
+
+social:
+    twitter:
+    linkedin:
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+
+Platform engineering has transitioned from an emerging trend in DevOps and infrastructure to a critical function for mid-to-large organizations. By 2026, Gartner predicts that 80% of software development companies will have internal platform services and dedicated teams to streamline development processes.
+One of the main goals in Platform engineering is to empower developers by providing self-service and autonomy while maintaining guardrails for security, compliance, and reliability. Achieving this balance requires integrating DevOps and security practices, referred to as DevSecOps best practices.
+During PulumiUP Europe 2024, the panel of industry experts explored this integration, including:
+
+- Jess Mink, Sr. Director of Platform Engineering at Honeycomb
+- Kief Morris, Global Head of Infrastructure Engineering at ThoughtWorks
+- Lindsay Jack, VP of Engineering & Security at Snyk
+- Nariman Aga-Tagiyev, Application Security Architect at WiseFrog Security
+- Komal Ali, Engineering Manager at Pulumi
+  
+This discussion centered on the core pillars of platform engineering, strategies for aligning DevOps and security, and overcoming challenges to build scalable, secure platforms.
+
+<!--more-->
+
+## In this article:
```

Comment:
Do we have a table of contents template in Hugo? If so, suggest utilizing it. If not, link to page sections; e.g. pulumi.com/blog/integrating-devops-and-security-for-scalable-platform-engineering/#the-core-pillars-of-platform-engineering in the first bullet "The Core Pillars of Platform Engineering"

Reply:
I wish there were a "built-in" nav generator, but no. I usually manually link them once I am sure each section's name stays as it is because it is a hassle. I am the only one who does it, though.